sources/packages/backend/apps/api/src/app.system-access.module.ts

```diff
@@ -65,11 +64,10 @@ import { DisbursementReceiptRequestService } from "./esdc-integration/disburseme
 import { FedRestrictionProcessingService } from "./esdc-integration/fed-restriction-integration/fed-restriction-processing.service";
 import { FedRestrictionIntegrationService } from "./esdc-integration/fed-restriction-integration/fed-restriction-integration.service";
 import { WorkflowClientService, SequenceControlService } from "@sims/services";
-import { IER12FileService } from "./institution-integration/ier-integration/ier12-file.service";
-import { IER12IntegrationService } from "./institution-integration/ier-integration/ier12-integration.service";
-
+import { IntegrationModule } from "@sims/integration";
+import { SshService } from "@sims/integration/services";
```

Comment:
Can we add a to-do here to remove the `SshService` once all integrations are moved?